test/mpi/main.cc

```diff
@@ -470,6 +471,7 @@ int main(int argc, char *argv[])
         }
     }
     test->create_teams(teams);
+    test->create_teams(teams, true);
```

Comment:
pls add command line argument to disable onesided tests

Reply:
done